sdk/include/psdk/shobjidl.idl

```diff
@@ -1605,6 +1605,11 @@ cpp_quote("#define CMF_CANRENAME     0x00000010")
 cpp_quote("#define CMF_NODEFAULT     0x00000020")
 cpp_quote("#define CMF_INCLUDESTATIC 0x00000040")
 cpp_quote("#define CMF_EXTENDEDVERBS 0x00000100")
+cpp_quote("#define CMF_DISABLEDVERBS 0x00000200")
+cpp_quote("#define CMF_ASYNCVERBSTATE    0x0400")
+cpp_quote("#define CMF_OPTIMIZEFORINVOKE 0x0800")
+cpp_quote("#define CMF_SYNCCASCADEMENU   0x1000")
+cpp_quote("#define CMF_DONOTPICKDEFAULT  0x2000")
```

Comment:
```suggestion
cpp_quote("#define CMF_ASYNCVERBSTATE    0x00000400")
cpp_quote("#define CMF_OPTIMIZEFORINVOKE 0x00000800")
cpp_quote("#define CMF_SYNCCASCADEMENU   0x00001000")
cpp_quote("#define CMF_DONOTPICKDEFAULT  0x00002000")
```

Reply:
Really? No pretty align? Keep in mind that these flags will never go beyond 0xffff.

Reply:
If you want pretty align, then align the other flags :rofl: (and I'm not talking about tab-"aligning" the values themselves, once you remove the `cpp_quote("` part)

Sp yeah, I wasn't looking at the "pretty align", but more about consistency of the values.

Reply:
If they don't go beyonf 0xffff then please shrink the others instead of expanding the new with extra zeroes.

Reply:
They don't but there is CMF_RESERVED 0xffff0000.

Reply:
I see. then zero-pad ofc.